py3-attrs.yaml

```diff
@@ -40,10 +45,27 @@ pipeline:
     runs: |
       git fetch --unshallow
 
-  - name: Python Install
-    runs: export SOURCE_DATE_EPOCH=$(git log -1 --format=%ct || $SOURCE_DATE_EPOCH) && pip3 install . --root=${{targets.destdir}}
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}}
+    dependencies:
+      runtime:
+        - python-${{range.key}}
+    pipeline:
+      - runs: |
+          export SOURCE_DATE_EPOCH=$(git log -1 --format=%ct || $SOURCE_DATE_EPOCH)
+          python${{range.key}} -m pip install --no-deps --no-build-isolation . --root=${{targets.contextdir}}
```

Comment:
You don't' need to add a runtime pytho ndep

Don''t do manual install like this use

```
    dependencies:
      provides:
        - py3-${{vars.pypi-package}}
      provider-priority: ${{range.value}}
    pipeline:
      - uses: py/pip-build-install
        with:
          python: python${{range.key}}
      - uses: strip
```